.travis.yml

```diff
@@ -16,8 +16,5 @@ matrix:
     env: PUPPET_GEM_VERSION="~> 3.0" FUTURE_PARSER="yes"
   - rvm: 1.9.3
     env: PUPPET_GEM_VERSION="3.3.0"
-  allow_failures:
-    - rvm: 2.1.5
-      env: PUPPET_GEM_VERSION="~> 4.0"
```

Comment:
Yes. Yes it did.  I need to modify modysnc settings to try and import this stuff